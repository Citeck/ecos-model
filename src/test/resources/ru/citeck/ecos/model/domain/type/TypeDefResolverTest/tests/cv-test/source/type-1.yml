---
id: type-1
parentRef: emodel/type@type-0

inheritCreateVariants: true

createVariants:
  - id: type-1-cv
    formRef: uiserv/form@abcdef
