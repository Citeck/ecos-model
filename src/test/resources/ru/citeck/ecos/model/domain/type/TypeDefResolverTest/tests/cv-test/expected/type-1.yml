---
id: type-1
name: type-1
sourceId: emodel/type-0-src
parentRef: emodel/type@type-0
metaRecord: emodel/type-0-src@

createVariants:
  - id: type-1-cv
    formRef: uiserv/form@abcdef
    name: {"en":"type-1"}
    sourceId: emodel/type-0-src
    typeRef: emodel/type@type-1
